docs/manual/docs/cmd/yammer/message/message-like-set.md

```diff
@@ -0,0 +1,47 @@
+# yammer message like set
+
+Adds or removes a like from a message
+
+## Usage
+
+```sh
+yammer message like set [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`--id <id>`|The Message ID to process
+`--enable`|Add or remove a like from a message. Default true
```

Comment:
Should be `--enable [enable]` to indicate that the option expects a value rather than being a switch